kanidmd/idm/src/valueset.rs

```diff
@@ -1058,17 +1058,11 @@ impl ValueSet {
             I::Utf8(set) => set.iter().cloned().collect(),
             I::Iutf8(set) => set.iter().cloned().collect(),
             I::Iname(set) => set.iter().cloned().collect(),
-            I::Uuid(set) => set
-                .iter()
-                .map(|u| u.as_hyphenated().to_string())
-                .collect(),
+            I::Uuid(set) => set.iter().map(|u| u.as_hyphenated().to_string()).collect(),
```

Comment:
What version of rust are you using? This kind of formatting back-and-forth I've seen before when there are different versions ....

Reply:
Oh I'm running 1.60, I'll push back to 1.59 and see what it shows.

Reply:
I'm on 1.60 too, so that's extra haunted.

Reply:
I went back to 1.56 and it didn't want me to change anything so... meh? Leaving this alone for now.